app/admin/announcements.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register Announcement do
+  config.sort_order = 'created_at_desc'
+
+  actions :index, :show, :new, :create
+  permit_params :message
```

Comment:
```suggestion
  permit_params :message
```

If this is not defined it'll use the policy `permitted_attributes`

Reply:
this comment confuses me haha. is there a suggested change here?

Reply:
LOL. Sorry, I've updated it!

Reply:
lol, thanks!